extensions/blocks/podcast-player/style.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Frontend (site) styles for Podcast Episodes
+ */
+
+.wp-block-jetpack-podcast-player {
+    border: 1px solid;
+}
+
+.wp-block-jetpack-podcast-player audio {
+    display: none;
```

Comment:
This will make the audio element inaccessible to screenreaders. Is that what you want to do here? If not then consider a method to visually hide without doing so for screenreaders.